parsing/parsetree.mli

```diff
@@ -441,6 +435,17 @@ and expression_desc =
   | Pexp_extension of extension  (** [[%id]] *)
   | Pexp_unreachable  (** [.] *)
 
+and expr_poly =
```

Comment:
Why not call this `method_body`? Do we expect this type to be reused in other contexts? If sticking with this name, for uniformity with the surrounding code, it should be called `expression_poly` or `poly_expression` (ie avoiding the abbreviation).

Reply:
Seeing how this pattern only occurs inside methods `method_body` is indeed a much nicer name. I'll change it